src/jdk.attach/windows/native/libattach/VirtualMachineImpl.c

```diff
@@ -231,16 +231,14 @@ JNIEXPORT jlong JNICALL Java_sun_tools_attach_VirtualMachineImpl_openProcess
     }
 
     /*
-     * On Windows 64-bit we need to handle 32-bit tools trying to attach to 64-bit
-     * processes (and visa versa). X-architecture attaching is currently not supported
-     * by this implementation.
+     * On Windows we need to handle 32-bit tools trying to attach to 64-bit
+     * processes, which is currently not supported by this implementation.
```

Comment:
It's the only way around. The tool uses the API attach so it's 64-bit and you want to catch the case where it attempts to attach a 32-bit VM.